api/GlobalEventHandlers.json

```diff
@@ -597,55 +597,6 @@
           }
         }
       },
-      "onblur": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/onblur",
-          "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#handler-onblur",
```

Comment:
Can you add these spec URLs for the removed entries to the existing entries that will stay?